typedmodels/models.py

```diff
@@ -245,40 +245,79 @@ def _model_has_field(cls, base_class, field_name):
     def _patch_fields_cache(cls, base_class):
         orig_get_fields = cls._meta._get_fields
 
-        def _get_fields(
-            self,
-            forward=True,
-            reverse=True,
-            include_parents=True,
-            include_hidden=False,
-            seen_models=None,
-        ):
-            cache_key = (
-                forward,
-                reverse,
-                include_parents,
-                include_hidden,
-                seen_models is None,
-            )
+        if django.VERSION >= (5, 0):
```

Comment:
Change necessary due to the change in signature of `Model._meta._get_fields`. It seems the simplest to have two versions split by version. If you use django-upgrade it can automatically drop the versioned block when you stop supporting Django < 5.0: https://github.com/adamchainz/django-upgrade/#versioned-blocks